opentelemetry-user-events-metrics/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "opentelemetry-user-events-metrics"
+version = "0.1.0"
+description = "Metrics exporter for user events"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-user-events-metrics"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-user-events-metrics"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["opentelemetry", "metrics", "user-events"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.60"
+autotests = false
+
+[lib]
+doctest = false
+
+[dependencies]
+opentelemetry_api = { version = "0.19", path = "../opentelemetry-api", features = ["metrics"] }
+opentelemetry_sdk = { version = "0.19", path = "../opentelemetry-sdk", features = ["metrics", "rt-tokio"] }
+tokio = { version = "1.0", features = ["full"] }
+protobuf = { version = "2.18", optional = true }
```

Comment:
is this optional?

Reply:
Removed because we already have opentelemetry-proto dependency.